tests/libcxx/expected_results.txt

```diff
@@ -519,7 +519,7 @@ std/utilities/meta/meta.trans/meta.trans.other/aligned_storage.pass.cpp SKIPPED
 # Compiler bug: VSO-106654 "error C2580 "multiple versions of a defaulted special member functions are not allowed" is bogus and ungrammatical"
 std/utilities/tuple/tuple.tuple/tuple.cnstr/test_lazy_sfinae.pass.cpp:0 SKIPPED
 
-# Compiler bug: VSO-406936 "is_constructible<int&&, double&> and is_constructible<const int&, ExplicitTo<int&&>> should be true"
+# Compiler bug: VSO-752709 "[Feedback] Constructing rvalue reference from non-reference-related lvalue reference"

```

Comment:
Why link to this internal feedback ticket rather than the public DevCom bug to which it corresponds?
```suggestion
# Compiler bug: DevCom-409222 "Constructing rvalue reference from non-reference-related lvalue reference"
```

Reply:
Done, thanks.